### PR TITLE
Fix missing DIL for new query server

### PR DESCRIPTION
**Pull request by alexet**
This makes the new queryserver write the DIL to the DIL path which should make it work. It also fixes the error message to report the DIL location.

I can't get extension development to work currently so I can't test this so whoever reviews this should test it.

### CODE REVIEW

File: extensions/ql-vscode/src/run-queries-shared.ts [hunk 1]
```diff
@@ -198,7 +198,7 @@ export class QueryEvaluationInfo {
         // This could be from the new query server
         // in which case we expect the qlo to be missing so we should ignore it
         throw new Error(
-          `DIL was not found. ${compiledQuery}`
+          `DIL was not found. ${this.dilPath}`
```
Comment (aeisenberg) [inline extensions/ql-vscode/src/run-queries-shared.ts:201]:
```suggestion
          `DIL was not found. Expected location: '${this.dilPath}'`
```

**Comment by hmakholm** [comment]
I suspect a CLI change will also be necessary, since `QueryServer2Command` never calls `CompilationCommon.setIncludeDILinQLO(true)`.

**Comment by alexet** [comment]
Yeah Henning is correct. I have managed to test this and it tells the query server to produce DIL but the new query server produces an empty file.

**Review by aeisenberg — APPROVED** [review]
This looks right to me.

File: extensions/ql-vscode/src/run-queries-shared.ts [hunk 2]
```diff
@@ -198,7 +198,7 @@ export class QueryEvaluationInfo {
         // This could be from the new query server
         // in which case we expect the qlo to be missing so we should ignore it
```
Comment (aeisenberg) [inline extensions/ql-vscode/src/run-queries-shared.ts:199]: This comment is no longer relevant.

Comment (alexet) [reply extensions/ql-vscode/src/run-queries-shared.ts:199]: No it is a still correct. The point was that if we are using the new query server we don't expect a qlo so we shouldn't report that the qlo is missing and report that the DIL is missing. If we are still using the old queryserver then a missing qlo is likely to be the problem.

**Pull request merged**